pkg/server/internal_server.go

```diff
@@ -21,6 +21,8 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.StringVar(&cfg.Config.HTTPTLSConfig.TLSKeyPath, "internal-server.http-tls-key-path", "", "HTTP internal server key path.")
 	f.StringVar(&cfg.Config.HTTPTLSConfig.ClientAuth, "internal-server.http-tls-client-auth", "", "HTTP TLS Client Auth type.")
 	f.StringVar(&cfg.Config.HTTPTLSConfig.ClientCAs, "internal-server.http-tls-ca-path", "", "HTTP TLS Client CA path.")
+	f.StringVar(&cfg.Config.CipherSuites, "internal-server.http-tls-cipher-suites", "", "HTTP TLS Cipher Suites.")
+	f.StringVar(&cfg.Config.MinVersion, "internal-server.http-tls-min-version", "", "HTTP TLS Min Version.")
```

Comment:
Are there supposed to be two pairs of these flags?
```suggestion
	f.StringVar(&cfg.HTTPTLSConfig.CipherSuites, "internal-server.http-tls-cipher-suites", "", "HTTP TLS Cipher Suites.")
	f.StringVar(&cfg.HTTPTLSConfig.MinVersion, "internal-server.http-tls-min-version", "", "HTTP TLS Min Version.")
	
	f.StringVar(&cfg.GRPCTLSConfig.CipherSuites, "internal-server.http-tls-cipher-suites", "", "GRPC TLS Cipher Suites.")
	f.StringVar(&cfg.GRPCTLSConfig.MinVersion, "internal-server.http-tls-min-version", "", "GRPC TLS Min Version.")
```